PopPUNK/network.py

```diff
@@ -535,18 +538,34 @@ def constructNetwork(rlist, qlist, assignments, within_label,
             accessory or euclidean distance
         sparse_input (numpy.array)
             Sparse distance matrix from lineage fit
+        G_df (cudf or pandas data frame)
+            Two column source/destination data frame of integers
         previous_network (str)
             Name of file containing a previous network to be integrated into this new
             network
         previous_pkl (str)
             Name of file containing the names of the sequences in the previous_network
+        betweenness_sample (int)
+            Number of sequences per component used to estimate betweenness using
+            a GPU. Smaller numbers are faster but less precise [default = 100]
         use_gpu (bool)
             Whether to use GPUs for network construction
 
     Returns:
         G (graph)
             The resulting network
     """
+    
+    # Set up for GPU if specified
+    if use_gpu:
+        # Build edge pandas dataframe
+        if not gpu_lib:
+           sys.stderr.write('Unable to load GPU libraries; exiting\n')
+           sys.exit(1)
```

Comment:
Agree with you in most cases, which have now been switched in 2065a89 - exceptions are the network functions, where switching to CPUs either won't generate equivalently-formatted outputs (because of the different G data structures), or a different file to the one specified could be loaded. Let me know what you think.